test_demo_any.py

```diff
@@ -0,0 +1,27 @@
+from os import path as osp
+
+import numpy as np
+
+from model import DeepLab
+from utils import (Dataset, Iterator, save_load_means, subtract_channel_means, single_demo, read_image)
+
+if __name__ == '__main__':
+
+    demo_dir = 'data/demos/deeplab/MyImg/'
+    models_dir = 'data/models/deeplab/resnet_101_voc2012/'
+    model_filename = 'resnet_101_0.6959.ckpt'
+    image_filename='data/datasets/MyImg/JPEGImages/00.jpg'
```

Comment:
Please support multiple images in an image directory. A for loop will just work.

Reply:
I've done changes you proposed and committed in my repo. How do I proceed? Should I create a new pull request? (Sorry I' m new to github :)

Reply:
Thanks pinaxe1, no new pull request is required.